resources/assets/ts/actions/chat-actions.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ *    Copyright 2015-2018 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+// tslint:disable:max-classes-per-file
+import Message from 'models/chat/message';
+import DispatcherAction from './dispatcher-action';
+
+export class ChatChannelSwitchAction implements DispatcherAction {
+  constructor(public channelId: number) {
```

Comment:
Is there a reason this takes `number` instead of `Channel` like how the other actions accept `Message`?